code/modules/vore/eating/living_ch.dm

```diff
@@ -0,0 +1,138 @@
+					return FALSE
+
+				RTB.reagents.vore_trans_to_mob(TR, transfer_amount, CHEM_VORE, 1, 0, TB)
+
+				if(TG != user)
+					user.visible_message("<span class='notice'>[user] fills their [TB] with [RTB.reagent_name] from [TG]'s [RTB].</span>")
+				else
+					user.visible_message("<span class='notice'>[user] fills their [TB] with [RTB.reagent_name] from their [RTB].</span>")
+
+			else if(TR.receive_reagents == FALSE)
+				to_chat(user, "<span class='warning'>This person's prefs dont allow that!</span>")
+				return FALSE
+
+			else
+				var/obj/belly/TB = input("Choose which organ to transfer to") as null|anything in TR.vore_organs
+				if(!TB)
+					return FALSE
+
+				if(!TR.reagents.get_free_space())
+					to_chat(user, "<span class='notice'>[TR]'s [TB] is full!</span>")
+					return FALSE
+
+				RTB.reagents.vore_trans_to_mob(TR, transfer_amount, CHEM_VORE, 1, 0, TB)
+
+				if(TG == user)
+					user.visible_message("<span class='notice'>[user] fills [TR]'s [TB] with [RTB.reagent_name] from their [RTB].</span>")
+				else
+					user.visible_message("<span class='notice'>[user] fills [TR]'s [TB] with [RTB.reagent_name] from [TG]'s [RTB].</span>")
+
+				add_attack_logs(user,TR,"Transfered reagents from [TG]'s [RTB] to [TR]'s [TB]")	//Bonus for staff so they can see if people have abused transfer and done pref breaks
+
+
+		if("Stomachs")
+			var/mob/living/TR = input(user,"Choose who to transfer to","Select Target") as null|mob in view(user.loc,1)
+			if(!TR)  return
+
+			if(TR == user) //Proceed, we dont need to have prefs enabled for transfer within user
+				RTB.reagents.vore_trans_to_mob(TR, transfer_amount, CHEM_INGEST, 1, 0, null)
+				if(TG != user)
+					user.visible_message("<span class='notice'>[user] fills their stomach with [RTB.reagent_name] from [TG]'s [RTB].</span>")
+				else
+					user.visible_message("<span class='notice'>[user] fills their stomach with [RTB.reagent_name] from their [RTB].</span>")
+
+			else if(TR.receive_reagents == FALSE)
+				to_chat(user, "<span class='warning'>This person's prefs dont allow that!</span>")
+				return FALSE
+
+			else
+				RTB.reagents.vore_trans_to_mob(TR, transfer_amount, CHEM_INGEST, 1, 0, null)
+				if(TG == user)
+					user.visible_message("<span class='notice'>[user] fills [TR]'s stomach with [RTB.reagent_name] from their [RTB].</span>")
+				else
+					user.visible_message("<span class='notice'>[user] fills [TR]'s stomach with [RTB.reagent_name] from [TG]'s [RTB].</span>")
+
+				add_attack_logs(user,TR,"Transfered reagents from [TG]'s [RTB] to [TR]'s Stomach")	//Bonus for staff so they can see if people have abused transfer and done pref breaks
+
+		if("Containers")
+			var/list/choices = list()
+			for(var/obj/item/weapon/reagent_containers/rc in view(user.loc,1))
+				choices += rc
+			var/obj/item/weapon/reagent_containers/T = input(user,"Choose what to transfer to","Select Target") as null|anything in choices
+			RTB.reagents.vore_trans_to_con(T, 1, 1, 0)

```

Comment:
I think this might be the offending line. There is no 'transfer_amount' on it and instead there are 1s. I was only able to pour 1 unit into a beaker, maybe this might be it? I can try modifying it a bit.

Reply:
yeah youre right, it appears the code on that proc went back to the earlier version where I didnt have the transfer option set up yet. will fix now